ulid.go

```diff
@@ -372,6 +374,14 @@ func Timestamp(t time.Time) uint64 {
 		uint64(t.Nanosecond()/int(time.Millisecond))
 }
 
+// TimestampToTime converts Unix milliseconds in the format
+// returned by the Timestamp function to a time.Time.
+func TimestampToTime(ms uint64) time.Time {
```

Comment:
Why not call it just `Time`?

Reply:
Sure. Might make it difficult to phrase the doc comment pointing devs to this method as it's on the ULID method also called ```Time```. I'll have a go though.